run-platform.sh

```diff
@@ -13,6 +13,9 @@ yellow_text='\033[33m'
 # set -x/xtrace uses PS4 for more info
 PS4="$blue_text""${0}:${LINENO}: ""$default_text"
 
+# TODO: Change to commit where v2 feature flag is removed
+V2_WARNING_VERSION="v0.90.3"
```

Comment:
@chandrasekharan-zipstack Suggest to remove this in favor of generalized release warnings as commented below.